root/programs/C#/Tests/Public.Test/Public.Test.csproj.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <StartAction>Program</StartAction>
+    <StartProgram>C:\root\programs\C#\Tests\packages\NUnit.Runners.2.6.3\tools\nunit.exe</StartProgram>
```

Comment:
@yusukemaegawa 

I have considered that the path of the problems can occur when deploy too deep position.

Therefore, if the plug-in needs to be located in the under the workspace folder.
In this case, this has been addressed in the XCOPY.
see : http://opentouryo.osscons.jp/index.php?Jenkins%E6%A7%8B%E7%AF%89#sffe761b